src/api/spec/models/backend/file_spec.rb

```diff
@@ -0,0 +1,310 @@
+require 'rails_helper'
+# WARNING: If you change #file_exists or #has_file test make sure
+# you uncomment the next line and start a test backend.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe Backend::File, vcr: true do
+  let(:user) { create(:user, login: 'user') }
+  let(:fake_filename) { 'fake_filename' }
```

Comment:
No, this name doesn't go to the backend. Is more about reusing the string in the tests....